functions/jsonpath-value/1.0/function.json

```diff
@@ -0,0 +1,89 @@
+{
+  "description": "Allows to select fields from json/objects using Jsonpath: https://www.npmjs.com/package/jsonpath",
+  "label": "Jsonpath",
+  "category": "Misc",
+  "icon": {
+    "name": "ActionsIcon",
+    "color": "Orange"
+  },
+  "options": [
+    {
+      "info": "The data or object",
+      "name": "data",
+      "label": "Data",
+      "meta": {
+        "type": "Text",
```

Comment:
I think we should change this to type `Value` with the following kinds: JSON, OBJECT STRING?